WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.m

```diff
@@ -401,20 +401,11 @@ - (void)configureViewConstraints
                                                                       metrics:nil
                                                                         views:views]];
 
-    // ReplyTextView Constraints
-    [[self.replyTextView.leftAnchor constraintEqualToAnchor:self.tableView.leftAnchor] setActive:YES];
-    [[self.replyTextView.rightAnchor constraintEqualToAnchor:self.tableView.rightAnchor] setActive:YES];
-
-    self.replyTextViewBottomConstraint = [NSLayoutConstraint constraintWithItem:self.view
-                                                                      attribute:NSLayoutAttributeBottom
-                                                                      relatedBy:NSLayoutRelationEqual
-                                                                         toItem:self.replyTextView
-                                                                      attribute:NSLayoutAttributeBottom
-                                                                     multiplier:1.0
-                                                                       constant:0.0];
-    self.replyTextViewBottomConstraint.priority = UILayoutPriorityDefaultHigh;
```

Comment:
[Nit] I think the property `replyTextViewBottomConstraint` is no longer needed, maybe it should be removed?

Reply:
Good catch! Thank you!